Require raise-if-root gem to stop running as root.

brodygov opened the pull request:
**Why**:

For least privilege and privilege separation, it is a best practice to
run applications as a normal user rather than root / the superuser.

**How**:

This gem raises an exception if anyone attempts to run the app as root.

zachmargolis commented on config/application.rb:
```diff
@@ -1,6 +1,8 @@
 require File.expand_path('../boot', __FILE__)
 require 'rails/all'
 
+require 'raise-if-root'
```
this will be included as part of the `Bundler.require` below right?

brodygov replied:
Yeah I didn't really know what I was doing here. Bundler.require will require all the things?

zachmargolis replied:
Yup! It requires all the things per environment (for example if you had included this gem in the "test" group of the Gemfile, it would only require it being installed/built for test, etc etc)

monfresh commented:
@brodygov What's the status of this PR? Is it still valid? If so, do you have time to make the requested changes and resolve the conflicts? If not, I'd be happy to.

brodygov commented:
Thanks for the ping @monfresh, I rebased it and removed the explicit require. I'm not sure we ever did the work needed to be sure we're not running cron jobs or the like as root, so this isn't safe to merge until those changes happen.

monfresh commented:
@brodygov Any updates on this? Should we close this PR for now until the infrastructure changes are in place?

brodygov commented:
I believe this can actually be merged today as long as we don't want to roll back the `deploy/migrate` script, which was the last thing in identity-devops that was running the app as root now that https://github.com/18F/identity-devops/pull/859 is merged.

monfresh commented:
Sounds good. I don't think we have any plans to roll back the `deploy/migrate` script. I will merge and keep fingers crossed.
